pwiz_tools/Skyline/Model/AuditLog/AuditLogEntry.cs

```diff
@@ -427,16 +426,22 @@ public bool IsRoot
             get { return ReferenceEquals(this, ROOT); }
         }
 
-        private AuditLogEntry(SrmDocument document, DateTime timeStamp, string reason,
+        private AuditLogEntry(DateTime timeStamp, string reason,
```

Comment:
It's very nice how we don't have to pass around the document around anymore just to get the FormatVersion in the AuditLogEntry constructor. Originally I must've been using the document for other things in the constructor since otherwise I could've just passed around the format version.